lua/lspconfig/serve_d.lua

```diff
@@ -0,0 +1,26 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+<<<<<<< HEAD
```

Comment:
Feel like this shouldn't be there :upside_down_face:

Reply:
Oop, yes, I fixed it but didn't push it.